int/api/thyra.go

```diff
@@ -78,7 +78,16 @@ func ThyraRegistryHandler(params operations.ThyraRegistryParams) middleware.Resp
 
 //nolint:nolintlint,ireturn
 func ThyraHomeHandler(params operations.ThyraHomeParams) middleware.Responder {
-	content, err := contentReact.ReadFile(basePathReact + params.Resource)
+	content, err := contentReact.ReadFile(basePathReact + "home/" + params.Resource)
+	if err != nil {
+		return operations.NewThyraHomeNotFound()
+	}
+
+	return NewCustomResponder(content, contentType(params.Resource), http.StatusOK)
+}
+
+func ThyraPluginManagerHandler(params operations.ThyraPluginManagerParams) middleware.Responder {
+	content, err := contentReact.ReadFile(basePathReact + "plugin-manager/" + params.Resource)
```

Comment:
This code is duplicated a lot.

Reply:
the line is duplicated two times at 81 and 90 all the rest of the code is different.
And as you said: We won't create a function to avoid a single line . 

So i think it's unnecesary to factorize this at this moment.

Reply:
@0xMazout issue created here: https://github.com/massalabs/thyra-plugin-hello-world/issues/7